src/h2load.cc

```diff
@@ -366,9 +438,16 @@ Client::~Client() {
     worker->sample_client_stat(&cstat);
   }
   ++worker->client_smp.n;
+  
+  if (!measurement_calculation_done) {
+    std::cout << "Duration timeout not called on Client: " << id << std::endl;
+    duration_timeout_cb(worker->loop, &duration_watcher, 0);
+  }
 }
 
-int Client::do_read() { return readfn(*this); }
+int Client::do_read() { 
+  return readfn(*this); 
```

Comment:
Haha. Removed it. Actually I did it to introduce some checking function, but we did not follow that method of implementation at the end.